R-proj/R/rand_rotate.R

```diff
@@ -112,18 +90,41 @@ rand_rotate <- function(Inputs){
   dim_gen = 0
   m_gen = 0
   exact_zono = FALSE
+  ball_only = FALSE
+  sam_simplex = FALSE
+  sam_can_simplex = FALSE
+  sam_arb_simplex = FALSE
+  sam_ball = FALSE
+  sam_sphere = FALSE
   #-------------------#
   
-  Mat = vol_R(A, W, e, Cheb_ball, annealing, win_len, N, C, ratio, frac, ball_walk, delta,
-              Vpoly, Zono, exact_zono, gen_only, Vpoly_gen, kind_gen, dim_gen, m_gen, round_only, 
-              rotate_only, sample_only, numpoints, variance, coordinate, rounding, verbose)
+  # set timer
+  tim = proc.time()
```

Comment:
in general timings are not needed in function declaration, the user can benchmark them if needed. So if you agree remove any timings from declarations. You can leave them in tests.

Reply:
done